moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -128,9 +128,10 @@ void RobotState::copyFrom(const RobotState& other)
   if (dirty_link_transforms_ == robot_model_->getRootJoint())
   {
     // everything is dirty; no point in copying transforms; copy positions, potentially velocity & acceleration
-    memcpy(position_, other.position_, robot_model_->getVariableCount() * sizeof(double) *
-                                           (1 + ((has_velocity_ || has_acceleration_ || has_effort_) ? 1 : 0) +
-                                            ((has_acceleration_ || has_effort_) ? 1 : 0)));
+    memcpy(position_, other.position_,
+           robot_model_->getVariableCount() * sizeof(double) *
+               (1 + ((has_velocity_ || has_acceleration_ || has_effort_) ? 1 : 0) +
+                ((has_acceleration_ || has_effort_) ? 1 : 0)));
```

Comment:
these changes should not be here

Reply:
@felixvd Are you sure you are using the correct clang formatting?

Reply:
No, I ran it from inside Visual Studio Code. It probably wasn't the right version.